brainframe/cli/__init__.py

```diff
@@ -0,0 +1,2 @@
+__version__ = "0.2.0"
+version_tag = f"v{__version__}"
```

Comment:
Do we need this? I think this is kind of legacy way of how BrainFrame server works. In the server we did it because we always did it from the start.

I'd be fine doing without a version_tag, since it makes it confusing (when do we use version, when do we not), and just always tag things / name things by `x.x.x` without the `v`

Reply:
Okay, I'm chill with that